### ISSUE TITLE
Project Acoustics Doesn't Work on Game Map

### ISSUE THREAD

**Issue by mikethal** [microsoft/ProjectAcoustics#184]
I'm trying to use Project Acoustics in a project on UE 5.1.1. I'm having trouble getting the acoustic voxels to show up in a level despite following all of the directions in [this video](https://www.youtube.com/watch?v=3uocCX0AMIg). Because of this, Baking Acoustics doesn't make any changes to the sound.

Interestingly, none of these issues occur when I switch to a test map in the same project! Voxels show up just fine and acoustics are clearly working.

Does anyone know why this might be happening? For example, does something level streaming or sublevels make a difference? Happy to answer more questions if needed. Would love to use this plugin because it seems incredible!

**Comment by NoelCross** [comment]
@mikethal The first thing that you should check is making sure that you have geometry tagged.  In the objects tab, do you see a non-zero number for "Objects tagged for Geometry" in the Objects tab.  If the number is non-zero, then highlight the selected geometry in the World Outliner view by hitting the "Select Tagged" button to inspect all meshes that have been tagged.  If that all looks good, then proceed to the Probes tab and hit the "Calculate" button.  Once that calculate process is complete, you should be able to view the voxels that will be used for baking once you turn on the Render Voxels option on the AcousticsDebugRenderer actor.  If you don't see voxels at this point, then it is possible that the geometry that you tagged is something that we don't currently support.  Let us know the details of the geometry that is not getting voxelized and we can help figure out how to include that geometry by possibly making minor code modifications to the plugin.

**Comment by mikethal** [comment]
@NoelCross Thanks so much! It sounds like this may not be geometry you support or there's another issue. I've followed all of your steps above, but I'm still not getting voxels or probes.

What details can I provide you with to help? Thanks again!

**Comment by mikethal** [comment]
@NoelCross Interestingly, it's not working for me on the default UE Third Person Example Map either.

**Comment by NoelCross** [comment]
@mikethal let's first concentrate on the UE Third Person Example.  I created a new Third Person C++ project with UE5.3.

Once launched, I enabled the UE5.3 version of the PA plugin and restarted the project.

Next, I opened the Bake Acoustics mode tab.

On the Objects tab, I selected "Static Meshes" in the Bulk Selection Helpers section and hit the "Select" button.  I noticed that the SM_SkySphere is selected so in the Outliner, I ctrl-click on that actor and see that the Currently selected objects show a value of 26.

In the tagging section, with the Geometry option selected, I hit the "Tag" button and see the number of "Tagged objects for Geometry:" bump up to 26.

Next, I click on the floor (SM_Cube) to deselect everything other than the floor.  I switched the tagging mode to "Navigation", select the "Tag" button and then see that "Tagged objects for Navigation" is now showing the number 1.

Next, I switch to the Probes tab, select a Fine resolution and press the "Calculate" button.

Once the probe calculation is complete, I see the message "SUCCESS: NumProbes: 139".

Next, I select the "AcousticsDebugRenderer" in the Outliner and enable the "Render Voxels" checkbox.  The "Render Probes" checkbox is already enabled.

After following these steps, I'm seeing the following image on the stage.

<img width="829" alt="image" src="https://github.com/microsoft/ProjectAcoustics/assets/19439257/341b255a-4440-4c44-985d-4bbcf0ca6500">

I'm hoping that you will be able to try these steps and see the same result that I'm seeing.  Make sure that you have the Realtime setting enabled in the Viewport Options menu.  If you don't have that set, you will not see the debug vizualizations in the editor.

<img width="211" alt="image" src="https://github.com/microsoft/ProjectAcoustics/assets/19439257/26eb3b7a-67f1-435d-a2cf-567aed23fba5">
